diff --git a/node_modules/@nx/js/src/generators/release-version/release-version.js b/node_modules/@nx/js/src/generators/release-version/release-version.js
index dde916e..a9f3628 100644
--- a/node_modules/@nx/js/src/generators/release-version/release-version.js
+++ b/node_modules/@nx/js/src/generators/release-version/release-version.js
@@ -191,7 +191,7 @@ To fix this you will either need to add a package.json file at that location, or
                         const releaseTagPattern = options.releaseGroup.releaseTagPattern;
                         latestMatchingGitTag = await (0, git_1.getLatestGitTagForPattern)(releaseTagPattern, {
                             projectName: project.name,
-                        }, options.releaseGroup.releaseTagPatternCheckAllBranchesWhen);
+                        }, options.releaseGroup.releaseTagPatternCheckAllBranchesWhen, specifier === 'prerelease');
                         if (!latestMatchingGitTag) {
                             if (options.fallbackCurrentVersionResolver === 'disk') {
                                 if (!currentVersionFromDisk &&
